test/e2e/kubectl/kubectl.go

```diff
@@ -1727,134 +1573,6 @@ metadata:
 		})
 	})
 
-	ginkgo.Describe("Kubectl run deployment", func() {
-		var nsFlag string
-		var dName string
-
-		ginkgo.BeforeEach(func() {
-			nsFlag = fmt.Sprintf("--namespace=%v", ns)
-			dName = "e2e-test-httpd-deployment"
-		})
-
-		ginkgo.AfterEach(func() {
-			err := wait.Poll(framework.Poll, 2*time.Minute, func() (bool, error) {
-				out, err := framework.RunKubectl(ns, "delete", "deployment", dName, nsFlag)
-				if err != nil {
-					if strings.Contains(err.Error(), "could not find default credentials") {
-						err = nil
-					}
-					return false, fmt.Errorf("kubectl delete failed output: %s, err: %v", out, err)
-				}
-				return true, nil
-			})
-			framework.ExpectNoError(err)
-		})
-
-		/*
-			Release : v1.9
-			Testname: Kubectl, run deployment
-			Description: Command ‘kubectl run’ MUST create a deployment, with --generator=deployment, when a image name is specified in the run command. After the run command there SHOULD be a deployment that should exist with one container running the specified image. Also there SHOULD be a Pod that is controlled by this deployment, with a container running the specified image.
-		*/
-		framework.ConformanceIt("should create a deployment from an image ", func() {
```

Comment:
I'm surprised we had a conformance test using deprecated function... that shouldn't really have happened

Reply:
yeah, my bad I didn't catch it earlier :disappointed: